tools/CoqMakefile.in

```diff
@@ -536,6 +537,7 @@ clean::
 	$(HIDE)find . -name .coq-native -type d -empty -delete
 	$(HIDE)rm -f $(VOFILES)
 	$(HIDE)rm -f $(VOFILES:.vo=.vio)
+	$(HIDE)rm -f $(VDFILES) .vdfiles
```

Comment:
I think you don't need to list `$(VDFILES)` here; it's covered by `$(ALLDFILES)` above.

Reply:
Yup.